bin/zabbix-sender

```diff
@@ -4,6 +4,7 @@
 use strict;
 use warnings;
 
+use lib 'lib';
```

Comment:
I needed it for development to have bin/zabbix-sender use my Zabbix/sender.pm from development.  You can leave out this line.